types/src/eth.rs

```diff
@@ -23,6 +24,8 @@ pub struct BlockResult {
     pub execution_results: Vec<ExecutionResult>,
     #[serde(rename = "mptwitness", skip_serializing, default)]
     pub mpt_witness: Vec<SMTTrace>,
+    #[serde(rename = "storageTrace", skip_serializing)]
```

Comment:
should we
```suggestion
    #[serde(rename = "storageTrace", skip_serializing, default)]
```
？

Reply:
I think `storageTrace` has been definitely a field existed in blockResult so without `default` instruction we could detect some irregulation early.

Reply:
idk.

I proposed this becaseu I encountered an error here: https://github.com/scroll-tech/roller-rs/pull/165

```
thread 'types::test::codec_traces' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `storageTrace`", line: 1, column: 430)', src/types.rs:79:76
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
test types::test::codec_traces ... FAILED
[0m[91m   Compiling pest_generator v2.1.3
thread 'queue::test::fifo' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `storageTrace`", line: 1, column: 325)', src/queue.rs:40:84
test queue::test::fifo ... FAILED
```


There might be solution but I haven't took a close look

Reply:
Seralize feature has been added. This issue should be resolved.